app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'AuthorId'
+  has_many :comments, foreign_key: 'AuthorId'
+  has_many :likes, foreign_key: 'AuthorId'
+
+  def most_recent_posts
+    Post.where(AuthorId: id).last(3)
+  end
+
+  validates :Name, presence: true
+  validates :PostCounter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Please remove the validations, You will implement validations in the upcoming project.

Reply:
Ok, I am removing that.

Reply:
Done.